testing/tests/DevExpress.exporter/imageCreator.tests.js

```diff
@@ -2366,6 +2366,34 @@ QUnit.test('getData returns Blob when it supported by Browser', function(assert)
     });
 });
 
+QUnit.test('data has base64 format if useBase64 used(T1136337)', function(assert) {
+    const done = assert.async();
+    const _getBlob = imageCreator._getBlob;
+    const _getBase64 = imageCreator._getBase64;
+    const testingMarkup = '<svg></svg>';
```

Comment:
If u use it in test it's "testing" by default

```suggestion
    const markup = '<svg></svg>';
```